app.js

```diff
@@ -14,10 +14,9 @@ function Image(name, path) {
 }
```

Comment:
Lines 12 and 13 seem like they don't belong there. Perhaps they were supposed to be outside of the constructor?